gradle-plugin/src/main/java/com/squareup/anvil/plugin/AnvilPlugin.kt

```diff
@@ -37,6 +34,15 @@ import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 import java.io.File
 import java.util.concurrent.ConcurrentHashMap
 
+@Suppress("DEPRECATION")
```

Comment:
Since `gradle-plugin` is now in the main build, it now gets the "warnings as errors" behavior in CI.